src/tests/multihost/alltests/test_misc_proxy.py

```diff
@@ -0,0 +1,222 @@
+""" Automation of proxy provider suite
+
+:requirement: IDM-SSSD-REQ : Proxy Provider
+:casecomponent: sssd
+:subsystemteam: sst_idm_sssd
+:upstream: yes
+"""
+from __future__ import print_function
+import pytest
+import subprocess
+import ldap
+from sssd.testlib.common.utils import sssdTools, \
+    SSHClient, LdapOperations
+
+
+def execute_cmd(multihost, command):
+    """ Execute command on client """
+    cmd = multihost.client[0].run_command(command)
+    return cmd
+
+
+@pytest.mark.usefixtures('setup_sssd_krb',
+                         'create_posix_usersgroups',
+                         'sssdproxyldap',
+                         'install_nslcd',
+                         'misc_test',
+                         'netgroups')
+@pytest.mark.tier1_3
+class TestProxyMisc(object):
+    """
+    This is test case class for proxy provider suite
+    """
+    def test_bz1036758(self, multihost, backupsssdconf):
+        """
+        :title: Allow for custom attributes in RDN bz1036758
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1036758
+        :id: 10eb49a4-b252-11ec-87ea-845cf3eff344
+        :steps:
+          1. Add a local user
```

Comment:
Currently the user is already added by the fixture, so strictly speaking it is not part of the test.

Reply:
changed